tests/driver_srf02/README.md

```diff
@@ -2,9 +2,49 @@
 This is a manual test application for the SRF02 ultrasonic ranger driver.
 
 # Usage
+First you always need to initialize the sensor using the `init` shell command
+and the devices address in right-aligned, decimal format (as specified in the
+reference manual), e.g.: `init 224`.
 
+If successful (the shell will tell you...), you can sample distance data from
+the device, either once (`shoot`) or continuously every second (`sample`).
 
-After initialization, the sensor value is read periodically and printed to the STDOUT.
+This test application also allows to re-program the I2C address of SRF02
+devices. For this initialize the device with its current address and than change
+the to the new address using the `addr` shell command (e.g. `addr 228`). After
+calling the `addr` command, this application will automatically re-initialize
+the device with the new address, so it is usable right away.
 
```

Comment:
IMO it's a stupid hardware design that you can change the address (non volatile) but can't read it out again (except looking at the LED at startup). It could be useful to describe that quickly or refer to the datasheet